tests/test.js

```diff
@@ -1,49 +1,169 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount, shallow, render } from 'enzyme';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import QuestionsAndAnswers from '../client/app/components/questionsAndAnswers.jsx';
+import Header from '../client/app/components/header.jsx';
+import Question from '../client/app/components/question.jsx';
+import Avatar from '../client/app/components/avatar.jsx';
+import ShowAnswersButton from '../client/app/components/showAnswersButton.jsx';
+import AnswerSubmissionForm from '../client/app/components/answerSubmissionForm.jsx';
+import QuestionSubmissionForm from '../client/app/components/questionSubmissionForm.jsx';
+import Answers from '../client/app/components/answers.jsx';
+import Answer from '../client/app/components/answer.jsx';
```

Comment:
Maybe a stretch goal but you could condense these lines with something like this: https://stackoverflow.com/questions/29722270/import-modules-from-files-in-directory